packages/gnutls.rb

```diff
@@ -3,41 +3,49 @@
+  source_url 'https://www.gnupg.org/ftp/gcrypt/gnutls/v3.7/gnutls-3.7.9.tar.xz'
+  source_sha256 'aaa03416cdbd54eb155187b359e3ec3ed52ec73df4df35a0edd49429ff64d844'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.2-1_armv7l/gnutls-3.7.2-1-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.2-1_armv7l/gnutls-3.7.2-1-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.2-1_i686/gnutls-3.7.2-1-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.2-1_x86_64/gnutls-3.7.2-1-chromeos-x86_64.tpxz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.9_armv7l/gnutls-3.7.9-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.9_armv7l/gnutls-3.7.9-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.9_i686/gnutls-3.7.9-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnutls/3.7.9_x86_64/gnutls-3.7.9-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: 'c2aa4651e3fcde7acb408d581be30d33ac5a99aeacdfb7ed1aa332ffa08807d8',
-     armv7l: 'c2aa4651e3fcde7acb408d581be30d33ac5a99aeacdfb7ed1aa332ffa08807d8',
-       i686: '7e70441115c1d1d5930f61ab8f81b91484dac0ad7fce74601abcd497346a0497',
-     x86_64: '607851025aec31be1628da21af43c29307024cf5169bed7f34707504d0099824'
+    aarch64: '4a7ed8769bc725a52a49d2cf6a00ba6cf582cbbb2cb64fde92c4ff2de325ef7f',
+     armv7l: '4a7ed8769bc725a52a49d2cf6a00ba6cf582cbbb2cb64fde92c4ff2de325ef7f',
+       i686: '208ccbca3dda2b6a0fce1d915dbdca10bb12dc017268d74edc279a120f034add',
+     x86_64: 'acfaf2260b1fed8a1d9bcab454cdf0ad21433cd9ad81b02aacb4a71a8516f596'
   })
 
+  depends_on 'brotli' # R
+  depends_on 'ca_certificates' # L
   depends_on 'gcc' # R
   depends_on 'glibc' # R
   depends_on 'gmp' # R
-  depends_on 'libffi'
+  depends_on 'libffi' => :build
   depends_on 'libidn2' # R
   depends_on 'libtasn1' # R
   depends_on 'libunbound' # R
   depends_on 'libunistring' # R
   depends_on 'nettle' # R
+  depends_on 'openssl' # R
   depends_on 'p11kit' # R This package cannot be built statically.
-  depends_on 'trousers'
-  depends_on 'zlibpkg'
+  depends_on 'trousers' => :build
+  depends_on 'zlibpkg' => :build
+  depends_on 'zstd' # R
+
+  no_env_options
 
   def self.prebuild
     # Use IPv4 fallback if default connection fails.
     system "#{CREW_PREFIX}/sbin/unbound-anchor -a '#{CREW_PREFIX}/etc/unbound/root.key' || #{CREW_PREFIX}/sbin/unbound-anchor -4 -a '#{CREW_PREFIX}/etc/unbound/root.key'"
+    # Rebuild ca-certificates.
+    system "#{CREW_PREFIX}/bin/update-ca-certificates --fresh --certsconf #{CREW_PREFIX}/etc/ca-certificates.conf"
```

Comment:
Could the test failures be due to the way the certificates are generated?

Reply:
I don't think so at all. I put that line there after I noticed this error, as part of making sure that there were updated certs, which some builds (of other packages) will complain about.